vault/resource_database_secret_backend_connection.go

```diff
@@ -388,6 +390,15 @@ func databaseSecretBackendConnectionResource() *schema.Resource {
 				ConflictsWith: util.CalculateConflictsWith(dbBackendOracle, dbBackendTypes),
 			},
 
+			"redshift": {
+				Type:          schema.TypeList,
+				Optional:      true,
+				Description:   "Connection parameters for the redshift-database-plugin plugin.",
+				Elem:          connectionStringResource(&connectionStringConfig{}),
+				MaxItems:      1,
+				ConflictsWith: util.CalculateConflictsWith("redshift", dbBackendTypes),
```

Comment:
Use the constant here:

```suggestion
				ConflictsWith: util.CalculateConflictsWith(dbBackendRedshift, dbBackendTypes),
```

Reply:
Fixed in 20fcb20